src/vs/code/electron-main/window.ts

```diff
@@ -424,24 +424,36 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 			this.dispose();
 		});
 
-		// Prevent loading of svgs
-		this._win.webContents.session.webRequest.onBeforeRequest(null!, (details, callback) => {
-			if (details.url.indexOf('.svg') > 0) {
+		// Prevent loading of remote svgs
+		this._win.webContents.session.webRequest.onBeforeRequest((details, callback) => {
+			if (details.url.endsWith('.svg')) {
 				const uri = URI.parse(details.url);
-				if (uri && !uri.scheme.match(/file/i) && uri.path.endsWith('.svg')) {
+				if (uri && uri.scheme.toLowerCase() !== 'file') {
 					return callback({ cancel: true });
 				}
 			}
 
 			return callback({});
 		});
 
-		this._win.webContents.session.webRequest.onHeadersReceived(null!, (details, callback) => {
+		this._win.webContents.session.webRequest.onHeadersReceived((details, callback) => {
 			const responseHeaders = details.responseHeaders as Record<string, (string) | (string[])>;
-
 			const contentType = (responseHeaders['content-type'] || responseHeaders['Content-Type']);
-			if (contentType && Array.isArray(contentType) && contentType.some(x => x.toLowerCase().indexOf('image/svg') >= 0)) {
```

Comment:
/cc @mjbvz based on commit https://github.com/microsoft/vscode/commit/577105a72c08400f3311b92efefbe2a0fe23982c

Can you clarify if this was only intended to prevent loading svg from remote url or all urls ? Is it okay to allow loading for `file`, `vscode-file` protocol which we are using to load the app resources ?

Reply:
Remote resources were the main concern. We wanted to block svgs in hovers for example

Reply:
Thanks for confirming @mjbvz